PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -175,7 +183,8 @@ struct HfCandidateCreator2Prong {
 
       /// reject candidates not satisfying the event selections
       auto collision = rowTrackIndexProng2.template collision_as<Coll>();
-      const auto rejectionMask = getHfCollisionRejectionMask<centEstimator>(collision, centralityMin, centralityMax, useSel8Trigger, maxPvPosZ, useTimeFrameBorderCut);
+      float centrality{-1.f};
+      const auto rejectionMask = getHfCollisionRejectionMask<true, centEstimator>(collision, centrality, centralityMin, centralityMax, useSel8Trigger, useTimeFrameBorderCut, -maxPvPosZ, maxPvPosZ, 0, -1.f);
```

Comment:
```suggestion
      const auto rejectionMask = getHfCollisionRejectionMask<true, centEstimator>(collision, centrality, centralityMin, centralityMax, useSel8Trigger, useTimeFrameBorderCut, -zPvPosMax, zPvPosMax, 0, -1.f);
```
Better to unify the names with the utils.

Reply:
Done